config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -40,6 +40,22 @@ config LOG_DEFAULT_LEVEL
 
 endif
 
+config FLASH
```

Comment:
hmm, I don't see that the `config CHIP` definition selects anything in `nrf/subsys/matter/Kconfig`. I guess these settings are implyed by OpenThread, or IEEE802154. I believe, that they were added here because WIFI doesn't implied them like OpenThread.

Reply:
It doesn't. What I mean is that is should. If something is 99% required for Matter it should be `imply`-ied by it.